src/Views/LandingPage.js

```diff
@@ -1,3 +1,47 @@
+<<<<<<< HEAD
+import React, { useState } from 'react';
+import Form from './Form';
+
+function LandingPage() {
+  const [posts, setPosts] = useState([]);
+  const [status, setStatus] = useState('idle');
+
+  const onSearch = async (subreddit) => {
+    setStatus('loading');
+    const url = `https://www.reddit.com/r/${subreddit}/top.json`;
+    const response = await fetch(url);
+    const { data } = await response.json();
+    setPosts(data.children);
+    setStatus('resolved');
+  };
```

Comment:
Ah interesting. This wasn't actually part of this task. The landing page only shows an image of the heatmap

Reply:
It's also best to stick to the scope of the task. The header task was only about adding the header component.

Reply:
Yeah I apologize, I think I was just following along and didn't even think to pay attention to what I was adding where and on what page. I will be correcting this today/this weekend